docs/src/content/hardhat-runner/docs/guides/project-setup.md

```diff
@@ -2,7 +2,7 @@
 
 Hardhat projects are `npm` projects with the `hardhat` package installed and a `hardhat.config.js` file.
```

Comment:
Let's add this at the top:

https://github.com/NomicFoundation/hardhat/blob/next-version/docs/src/content/hardhat-runner/docs/getting-started/index.md?plain=1#L54-L58